src/modules/user/userSwagerDocs.js

```diff
@@ -1,478 +1,499 @@
+ *                     firstName:
+ *                       type: string
+ *                       description: The user's first name.
+ *                       example: John
+ *                     lastName:
+ *                       type: string
+ *                       description: The user's last name.
+ *                       example: Smith
+ *                     email:
+ *                       type: string
+ *                       format: email
+ *                       description: The user's email.
+ *                       example: example@example.com
+ *                     avatar:
+ *                       type: string
+ *                       format: URL
+ *                       description: Avatar URL.
+ *                       example: example.com/example.jpeg
+ *                     about:
+ *                       type: string
+ *                       description: Details about te user.
+ *                       example: "I'm a student of front-end....."
+ *                     skills:
+ *                       type: [string]
+ *                       description: Array of users skills.
+ *                       example: [HTML, CSS, JavaScript, Git, GitHub]
+ *                     applications:
+ *                       type: [object]
+ *                       description: An array of application objects
+ *                       example: [application]
+ *                     offers:
+ *                       type: [object]
+ *                       description: An array of offers objects
+ *                       example: [offer]
+ *                     listings:
+ *                       type: [object]
+ *                       description: An array of listing objects
+ *                       example: [listing]
+ *                     Created:
+ *                       type: string
+ *                       format: date
+ *                       description: Date user joined.
+ *                       example: 2023-01-16T19:55:13.609Z
+ *                     Updated:
+ *                       type: string
+ *                       format: date
+ *                       description: Most recent update.
+ *                       example: 2023-01-16T19:55:13.609Z
+ *       400:
+ *         description: Bad request, user id is undefined / Could not find user
+ *       500:
+ *         description: Internal server error
+ */
+
+//------------------ PUT /users/{id} --------------------
+/**
+ * @swagger
+ * /users/{id}:
+ *   put:
+ *     summary: Update user details.
+ *     tags: [Users]
+ *     description: Updates users details, for password and email changes the current password is required.
+ *     components:
+ *       BearerAuth:
+ *         type: jsonToken
+ *         in: header
+ *         scheme: bearer
+ *     parameters:
+ *       - in: path
+ *         name: id
+ *         required: true
+ *         description: String ID of the user to retrieve.
+ *         schema:
+ *           type: String

```

Comment:
We have that the id is required but we need the request body to be defined too for updating a user this was forgotten the first time around.
```js
 *     requestBody:
 *       required: true
 *       content:
 *         application/json:
 *           schema:
 *             type: object
 *             properties:
 *               email:
 *                 type: string (email)
 *                 description: The user's email.
 *                 example: example@example.com
 *               password:
 *                 type: string
 *                 description: The user's desired password.
 *                 example: Password123
 *               confirmpassword:
 *                 type: string
 *                 description: The user's current password.
 *                 example: Password123
 *               etc.......
```

Reply:
done